tests/unit/steps/test_steps.py

```diff
@@ -251,6 +251,17 @@ def test_step_add_step_failed():
         plan.add_step(MagicMock())
 
 
+def test_step_add_steps():
+    """Test BaseStep adding sub steps at once."""
```

Comment:
I think it would be nice to have a text showing that an empty step is not added

Reply:
What do you mean by that? Like in docstring of test? I'm not testing that empty step is not added.